web/tests/people.test.js

```diff
@@ -264,6 +264,15 @@ test_people("basics", () => {
     assert.equal(realm_persons.length, 1);
     assert.equal(people.get_realm_active_human_user_ids().length, 1);
 
+    assert.deepEqual(people.get_realm_active_human_users(), [me]);
+
+    let realm_humans = people.get_realm_active_human_users();
+    assert.equal(realm_humans[0].full_name, "Me Myself");
+
+    realm_humans = people.get_realm_active_human_users();
```

Comment:
Why is this duplicated? You can just call `get_realm_active_human_users` once and can check length and name.